### PR TITLE
feat: [PIE-4267]: Added cypress test case for environment configuration

### PR DESCRIPTION
**Pull request by Eric761**
##### Summary:

Added cypress test case for environment configuration

##### Jira Links:

https://harness.atlassian.net/browse/PIE-4267

##### Screenshots:

Screenshot 2022-06-30 at 8 30 43 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by Eric761** [comment]
retrigger titlecheck

**Comment by Eric761** [comment]
retrigger titlecheck

**Comment by Eric761** [comment]
retrigger jest
retrigger titlecheck

**Comment by Eric761** [comment]
retrigger jest
retrigger titlecheck

**Comment by Eric761** [comment]
retrigger jest

**Review by kapilkumawat86 — APPROVED** [review]

**Review by satyam-harness — APPROVED** [review]

File: cypress/integration/75-cd/EnvironmentConfiguration.spec.ts [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+  })
+
+  it('Environment Variables Addition & YAML/Visual parity', () => {
+    cy.intercept('GET', environmentConfigurationCall, {
+      fixture: 'ng/api/environmentConfiguration/environmentConfiguration.json'
+    })
+    cy.intercept('GET', environmentConfigurationSecretCall, {
+      fixture: 'ng/api/environmentConfiguration/createEnvironmentConfigurationSecret.json'
+    }).as('envSecretCall')
+    cy.wait(3000)
```
Comment (udayvunnam) [inline cypress/integration/75-cd/EnvironmentConfiguration.spec.ts:44]: visitPageAssertion already has a good timeout. cy.wait(3000) makes the test must wait for 3000ms. Do we need this?

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Pull request merged**